clang/lib/Parse/Parser.cpp

```diff
@@ -1196,9 +1196,20 @@ void Parser::ProcessPluginSyntax(ParsingDeclarator &D) {
   llvm::raw_string_ostream ReplacementOS(Replacement);
 
   ReplacementOS << "\n{\n";
-  SHI->second->GetReplacement(PP, D, Toks, ReplacementOS);
   ReplacementOS << "\n}\n";
```

Comment:
What happens if the function does not return void? Don't we get a warning that the function doesn't return an appropriate value?